Twitter/Twitter_Generate_leads_from_posts.ipynb

```diff
@@ -0,0 +1,329 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6ec59f5e-ac9e-4ab0-a0e9-51b01b38dcaa",
+   "metadata": {},
+   "source": [
+    "**Tags:** #twitter #post #comments #naas_drivers #automation"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c40e4f30-0b07-4a3b-b39b-7cb460053f74",
+   "metadata": {},
+   "source": [
+    "**Author:** [Alok Chilka](https://www.linkedin.com/in/calok64/)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "618b546f-943a-498c-9eb6-637a1e992f21",
+   "metadata": {},
+   "source": [
+    "## Input"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "91fc2ccf-9a27-40f4-bb62-df542a468bc6",
+   "metadata": {
+    "execution": {
+     "iopub.execute_input": "2022-01-08T05:48:50.458541Z",
+     "iopub.status.busy": "2022-01-08T05:48:50.458187Z",
+     "iopub.status.idle": "2022-01-08T05:48:50.465249Z",
+     "shell.execute_reply": "2022-01-08T05:48:50.464699Z",
+     "shell.execute_reply.started": "2022-01-08T05:48:50.458451Z"
+    }
+   },
+   "source": [
+    "### Import libraries"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e52f830d-57df-4d2d-9b34-35e431afd2c9",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "!pip install --user tweepy"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4f6774a6-ff96-4c39-87a7-69a1fd3777cc",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "from naas_drivers import linkedin, hubspot\n",
+    "import pandas as pd\n",
+    "import numpy as np\n",
+    "import naas\n",
+    "from datetime import datetime, timedelta\n",
+    "import requests\n",
+    "import json\n",
+    "import tweepy"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "958d3406-bf4c-48b9-b56b-ad4dad7caaae",
+   "metadata": {},
+   "source": [
+    "### Setup your Twitter"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "cb06a98e-6e82-4a9e-bedc-9bc7a6cb30dc",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "#Twitter cookies\n",
```

Comment:
Hello @calok64, it would be awesome to get some details on how to setup Twitter to get this bearer token I think. Do you think you could add some instructions?

Reply:
@Dr0p42 made the required changes for getting token and also added output as dataframe. Please review and let me know